moveit_simple/src/moveit_simple.cpp

```diff
@@ -62,7 +60,15 @@ Robot::Robot(const ros::NodeHandle & nh, const std::string &robot_description,
   return;
 }
 
+Robot::Robot(const ros::NodeHandle &nh, const std::string &robot_description,
+  const std::string &group_name, const std::string &ik_base_frame,
+  const std::string &ik_tip_frame) : Robot(nh, robot_description, group_name)
+{ 
+  ik_base_frame_ = ik_base_frame;
+  ik_tip_frame_ = ik_tip_frame;
 
+  ik_transforms_found_ = this->computeIKTransforms();
+}
```

Comment:
You should output a log message that indicates this special handling of base and tip frames.

Reply:
Done.